src/ConnectedMode/Migration/IMigrationPrompt.cs

```diff
@@ -34,7 +35,7 @@ namespace SonarLint.VisualStudio.ConnectedMode.Migration
     /// </summary>
     internal interface IMigrationPrompt : IDisposable
     {
-        Task ShowAsync();
+        Task ShowAsync(BoundSonarQubeProject oldBinding);
```

Comment:
The migration checker is now responsible for telling the prompt which project to bind to.